test-monitor/nitestmonitor.yml

```diff
@@ -1247,6 +1247,11 @@ paths:
           required: true
           schema:
             $ref: '#/definitions/TestStepCreateOrUpdateRequestObject'
+        - in: query
+          name: updateResultTotalTime
+          description: Determine test result total time from the test step total times
```

Comment:
If this is a new parameter being added, the description should include what version of the operation it was added in so clients can know if the server supports it or not.